### PR TITLE
Add ah_build module

### PR DESCRIPTION
**Pull request by Tompage1994**
### What does this PR do?
Adds a module to run an ansible-galaxy build. It pulls from the ansible modules which are used to run the ansible-galaxy CLI. 

### How should this be tested?
Try using 

```
ansible -m redhat_cop.ah_configuration.ah_build localhost -c local [-a "force=true path=/path/to/collection output_path=/path/to/out"]
```

instead of 

```
ansible-galaxy collection install [-f --output-path /path/to/out /path/to/collection]
```

### Is there a relevant Issue open for this?
N/A

### Other Relevant info, PRs, etc.
Linting will fail until #19 is merged. Please merge that first

### CODE REVIEW

**Comment by Tompage1994** [comment]
With this we will really be able to streamline the `publish` role to just use these modules rather than `command`

**Comment by Tompage1994** [comment]
Linting will fail until #19 is merged due to previous linting issues

File: plugins/modules/ah_build.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      required: False
+      type: str
+      default: "."
+
+"""
+
+
+EXAMPLES = """
+- name: Approve redhat_cop.ah_configuration:v1.0.0
+  ah_approval:
```
Comment (sean-m-sullivan) [inline plugins/modules/ah_build.py]: So two things here, lets change the approve to build, and remove the version, and change the example module to ah_build

**Review by sean-m-sullivan — APPROVED** [review]
LGTM

**Pull request merged**